src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
try not to use static as much as possible, static is a procedural-based programming style, not OOP
```suggestion
    public Color getRandomColor() {
```